From: Jakub Jirutka <jakub@jirutka.cz>
Date: Tue, 28 Jun 2022 22:36:16 +0200
Subject: [PATCH] Change default PATH to the Alpine's default

Use the same PATH as in openssh, sudo and our /etc/profile.

--- a/doas.c
+++ b/doas.c
@@ -238,8 +238,8 @@
 int
 main(int argc, char **argv)
 {
-	const char *safepath = "/bin:/sbin:/usr/bin:/usr/sbin:"
-	    "/usr/local/bin:/usr/local/sbin";
+	const char *safepath = "/usr/local/sbin:/usr/local/bin:"
+	    "/usr/sbin:/usr/bin:/sbin:/bin";
 	const char *confpath = NULL;
 	char *shargv[] = { NULL, NULL };
 	char *sh;
